src/infra/minio-secrets.yml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: minio-secrets
+type: Opaque
+data:
+  MINIO_URL: aHR0cHM6Ly9taW5pbzo5MDAw
+  MINIO_ACCESS_KEY: UTNBTTNVUTg2N1NQUVFBNDNQMkY=
+  MINIO_SECRET_KEY: enVmK3RmdGVTbHN3UnU3Qko4Nndla2l0bmlmSUxiWmFtMUtZWTNURw==
+  MINIO_BUCKET_NAME: ZXZlbnRz
```

Comment:
No need to encrypt  bucket name. Traceability of the project will be better, if you will use the configmap to store such information like name of the bucket.